### DISCUSSION TITLE
how do i create a login?

### DISCUSSION THREAD

**Discussion by constttdev** [pocketbase/pocketbase#5310] [Q&A]
so i want to create a login site and i have done the register with a friend but idk how to like do it so when i put in the right credentials in the login form that it prints logged in in console?

register: https://pastes.dev/9jYXF1wrce
login: https://pastes.dev/xFHLjvGsqe

**Comment by orenaksakal** [comment]
you can find it in the documentation and for specific collection documentation you can

1) go to YOUR_INSTANCE_URL_/#/collections?collectionId=_pb_users_auth_&filter=&sort=-created
2) click the Api Preview button on the top right to reveal documentation panel
